### DISCUSSION TITLE
A stupid question: Why the code-lines is negative?

### DISCUSSION THREAD

**Discussion by HackYardo** [GenieFramework/Genie.jl#512] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/57677458/161454264-1f706ebc-2f04-43ef-96ea-b9c4f81553ae.png)

Sorry for the stupid question, but why?

**Comment by HackYardo** [comment]
The numbers of code lines are from the Genie's contributors pages:
```html
https://github.com/GenieFramework/Genie.jl/graphs/contributors
``` 
I wonder how it could be negative?

**Reply by essenciary** [reply]
Yeah, something is weird :))

**Reply by AbhimanyuAryan** [reply]
aah ok

**Reply by chunjiw** [reply]
Lines from all contributors must be positive, but lines from a single contributor is possible to be negative. Imagine someone who only deleted some lines.

**Reply by essenciary** [reply]
Yes, but that doesn't make sense for my user :D

**Comment by HackYardo** [comment, marked as answer]
https://github.com/GenieFramework/Genie.jl/discussions/512#discussioncomment-2502670
Perhaps it is the cause.
